streaming/src/main/scala/org/apache/spark/streaming/StreamingContext.scala

```diff
@@ -114,11 +123,15 @@ class StreamingContext private[streaming] (
 
   private[streaming] val isCheckpointPresent = (cp_ != null)
 
+  private[streaming] val isSparkContextPresent = (sc_ != null)
```

Comment:
This seems to have been defined for setting `sc` below, but once `sc` is set this variable can be false, while `sc` is no longer `null`. This `val` should probably not be added, else we risk ending up having bugs caused by this being `false` which it can never be, since a spark context is always present.